experimenter/experimenter/experiments/constants.py

```diff
@@ -813,7 +813,8 @@ class EmailType(models.TextChoices):
     WARNING_PREF_FLIPS_PREF_CONTROLLED_BY_FEATURE = (
         "Pref '{pref}' is controlled by a variable in feature {feature_config_slug}'"
     )
-    ENROLLMENT_END = "Enrollment End"
+    OBSERVING = "Observing"
+    ENROLLING = "Enrolling"
```

Comment:
I'd suggest that "Enrolling" and "Observing" be changed to "Enrollment" and "Observation", respectively, to make them seem more like states that are not necessarily current. This one's definitely a personal preference though so if others feel differently that's fine too.

Reply:
@jaredlockhart what do you think?

Reply:
Oh yeah that's not a bad point.

Draft
Preview
Review
Enrollment
Observation
Complete


Yeah I think that makes sense 👍